### PR TITLE
First pass of change of purpose

### PR DESCRIPTION
**Pull request by torgo**
This modifies the first sentences of the Purpose section to address #79.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -120,9 +120,8 @@
 
 ### Purpose {#purpose}
 
-This document is intended to guide thoughtful development of
-web platform technologies,
-for standards developers and website authors alike.
+This is a statement of the ethical principles of the w3c community.
```
Comment (torgo) [inline index.html]:
```suggestion
This is a statement of the ethical principles of the W3C community. 
```

Comment (rhiaro) [inline index.html]:
```suggestion
This is a statement of the ethical principles of the W3C community. 
```

**Review by hadleybeeman — APPROVED** [review]

**Review by rhiaro — APPROVED** [review]

**Comment by mnot** [comment]
This document is a Note, not a consensus document. As such this is probably going too far...

**Comment by torgo** [comment]
Hi @mnot  this was added in anticipation that we are moving this through the Statement track.

**Pull request merged**